examples/mugen/retrieval/s3d.py

```diff
@@ -30,17 +30,17 @@ def __init__(self, num_class):
             BasicConv3d(64, 64, kernel_size=1, stride=1),
             SepConv3d(64, 192, kernel_size=3, stride=1, padding=1),
             nn.MaxPool3d(kernel_size=(1, 3, 3), stride=(1, 2, 2), padding=(0, 1, 1)),
-            Mixed3b(),
-            Mixed3c(),
+            MixedConvsBlock(192, 64, 96, 128, 16, 32, 32),
+            MixedConvsBlock(256, 128, 128, 192, 32, 96, 64),
             nn.MaxPool3d(kernel_size=(3, 3, 3), stride=(2, 2, 2), padding=(1, 1, 1)),
-            Mixed4b(),
-            Mixed4c(),
-            Mixed4d(),
-            Mixed4e(),
-            Mixed4f(),
+            MixedConvsBlock(480, 192, 96, 208, 16, 48, 64),
+            MixedConvsBlock(512, 160, 112, 224, 24, 64, 64),
+            MixedConvsBlock(512, 128, 128, 256, 24, 64, 64),
+            MixedConvsBlock(512, 112, 144, 288, 32, 64, 64),
+            MixedConvsBlock(528, 256, 160, 320, 32, 128, 128),
             nn.MaxPool3d(kernel_size=(2, 2, 2), stride=(2, 2, 2), padding=(0, 0, 0)),
-            Mixed5b(),
-            Mixed5c(),
+            MixedConvsBlock(832, 256, 160, 320, 32, 128, 128),
+            MixedConvsBlock(832, 384, 192, 384, 48, 128, 128),
```

Comment:
Sorry putting a comment on a random position because Github doesn't let me comment on the constructor of the `S3D` class. We need to align the arguments you pass to make them similar to what TorchVision is currently using (see [Video Resnet](https://github.com/pytorch/vision/blob/main/torchvision/models/video/resnet.py#L197) class, nothing to different).